.github/workflows/ci.yml

```diff
@@ -690,12 +687,10 @@ jobs:
         if: ${{ github.event_name != 'pull_request' }}
       - uses: actions/setup-node@49933ea5288caeca8642d1e84afbd3f7d6820020 # v4
         with:
-          registry-url: "https://registry.npmjs.org"
           node-version-file: ".nvmrc"
-      - name: Install npm
-        run: npm i -g npm@11.6.0
-        shell: bash
-      - run: npm ci
+      - name: Install pnpm
+        uses: pnpm/action-setup@fe02b34f77f8bc703788d5817da081398fad5dd2 #v4.0.0
```

Comment:
Do we need to specify which version of `pnpm` to install?

Or does that `action-setup` ties it to a specific pnpm version?